Bottom: 6292bf4332fed13def9d05c260d9536652256d02
Top:    98c29b95ca6fab8d93e0da0884b8919aaa38d7aa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-07 18:01:32 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
